core/commands/IssueAddCommand.php

```diff
@@ -332,7 +332,16 @@ protected function process() {
 
 		# Add Tags
 		if( isset( $t_issue['tags'] ) && is_array( $t_issue['tags'] ) ) {
-			mci_tag_set_for_issue( $t_issue_id, $t_issue['tags'], $this->user_id );
+			$t_tags = array();
+			foreach( $t_issue['tags'] as $t_tag ) {
+				if( $t_tag['id'] === -1 ) {
+					$t_tag['id'] = tag_create( $t_tag['name'] );
```

Comment:
You could use `$this->user_id` as additional parameter to `tag_create`

Reply:
Added user_id to the call.